shard/src/emoji_manager.py

```diff
@@ -412,11 +412,16 @@ async def emojis(self, ctx):
                 logger.debug("file is good")
                 message = "Enclose the name (case sensitive) of cached emoji in `:`s to auto-load it into a message"
                 # msg = await ctx.send(message, file=discord.File(file, "cool.png"))
-                data, _ = await self.bot.manager_client.publish_file(
-                    data=base64.b64encode(file.getvalue()).decode('ascii'))
+                try:
+                    data = await self.bot.manager_client.publish_file(
+                        message_type.File(file=file.getvalue()))
+                except Exception:
+                    logger.info(f"Shard {self.bot.shard_id} failed to upload emoji")
+                    await ctx.send("Failed to upload emoji file")
```

Comment:
make the error "Unable to generate cached emoji previews" or something that would make more sense to the user